docs/examples/objc/iOS-Carthage/ThemisTest.xcodeproj/project.pbxproj

```diff
@@ -258,6 +266,8 @@
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				EXCLUDED_ARCHS = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
```

Comment:
Does this mean that examples require Xcode 12+ to compile?

Reply:
Yes. I understood that Xcode 11 is not compatible with Xcode 12. Basically, Themis 0.13.0 works with Xcode 11, Themis 0.13.3 works with Xcode 12, and that's all

Reply:
Dang, I guess I'll have to update to 10.15 after all :joy:

Reply:
I believe so. I will check if I can do something with this.

Reply:
Okay, so for now we will leave it as is because there no nice solution to support both Xcodes